frontend/javascripts/admin/job/job_list_view.tsx

```diff
@@ -157,14 +157,20 @@ function JobListView() {
     setSearchQuery(event.target.value);
   }
 
+  function getLinkToDataset(job: APIJob) {
+    // prefer updated link over legacy link.
+    if (job.datasetId != null)
+      return `/datasets/${getReadableURLPart({ name: job.datasetName || "unknown_name", id: job.datasetId })}/view`;
+    if (job.organizationId != null && job.datasetName)
```

Comment:
shouldn't it be:
```suggestion
    if (job.organizationId != null && (job.datasetName != null || job.datasetDirectoryName != null))
```

?

Reply:
good point! :sweat_smile: